### ISSUE TITLE
libCamera make messy :)

### ISSUE THREAD

**Issue by MyraBaba** [Qengineering/Traffic-Counter-RPi_64-bit#1]
Hi,

If there is a good-old days version of the opencv and camera without libcamera would be perfect. bulseye make life harder and ask more cpu :)

thanks

**Comment by Qengineering** [comment]
@MyraBaba ,

I agree with you, LibCamera isn't the most wise decision the Raspberry Pi foundation made.
However, if you don't want frame buffers to overflow, or large inference times, you need to use a streamer.
Both, GStreamer and ffmpeg use more CPU than LCCV. Where LCCV uses 10%, GStreamer needs 25%. See [here](https://github.com/Qengineering/LCCV#libcamera-c-api-wrapper-for-opencv-on-a-raspberry-pi-4-with-64-bit-bullseye-os)
That's why I use Bullseye and LCCV. 
Feel free to modify the code for Buster and GStreamer. Given the repo's here, it can't be that difficult.

**Comment by MyraBaba** [comment]
@Qengineering  I am modifying let you know.

what will be the best model for heavy intersection road not just like vertical one and also small vehicles? 
I tried Yolov7 with large model , quite good but computational gpu expensive.

**Comment by Qengineering** [comment]
YoloX or YoloV7 are the best. As you mentioned, you must have a lot of computational power when employed.
